How to enter DLC step added

HappyVlane opened the pull request:
[Pull Request Web Preview](https://happyvlane.github.io/dark-souls-3-cheat-sheet/)

Please don't merge this just yet. Want to polish it a bit first and I'd rather not create an additional pull request. Thanks.
# Changes

First step is now a how-to on entering the DLC

HappyVlane committed:
How to enter DLC step added

First step is now a how-to on entering the DLC

ZKjellberg commented:
Sorry to make you offset all the IDs, I realize that is a tedious task. Thanks for adding my input. I'll wait for your word on when to merge. 👍

ZKjellberg commented:
Is this ready to merge? I don't see any further activity.

HappyVlane committed:
Added additional steps to the DLC area + vanilla NPC interaction

- Some items where missing in the DLC (a soul, Blessed Gem and Ember)
- Added interaction with Eygon of Carim for Dragonslayer

HappyVlane commented:
Okay, just finished the third playthrough and added some missing items plus an interaction with Eygon in the base game. Ready to merge now.

Also noticed a possible error with the wyvern in Archdragon Peak regarding the amount of souls received, but would have to check it again. Gonna check it in my next run.

ZKjellberg commented:
I've merged your changes. I just discovered a glitch with Offline Application Cache causing the HTTP version to not update. I am going to create an issue. I want people to migrate over to HTTPS, so I might put up a banner to start a migration and implement the code to force switch later.
